tasks/main.yml

```diff
@@ -58,16 +60,19 @@
     src: manage_certs.conf
     dest: "{{ LOAD_BALANCER_MANAGE_CERTS_CONF }}"
 
-- name: copy config watcher systemd service file
+- name: copy config and cert watcher systemd service files
   template:
-    src: haproxy-config-watcher.service
-    dest: /lib/systemd/system/haproxy-config-watcher.service
-
-- name: enable and start config watcher
-  command: systemctl {{ item }} haproxy-config-watcher
+    src: "{{ item }}"
+    dest: "/lib/systemd/system/{{ item }}"
   with_items:
-    - enable
-    - restart
+    - haproxy-config-watcher.service
+    - cert-watcher.service
+
+- name: enable and start config and cert watcher
+  command: systemctl {{ item | join(" ") }}
```

Comment:
Brandon had the same comment, so it's time to add a comment to the code.

We can probably migrate to the latest Ansible version without any issues, but I'm currently too busy to try.